measures/ApplyUpgrade/tests/apply_upgrade_test.rb

```diff
@@ -129,7 +129,9 @@ def _test_retaining_capacities(osw_file, args_hash, expected_capacities)
     runner = OpenStudio::Measure::OSRunner.new(OpenStudio::WorkflowJSON.new)
 
     # Apply measure
+    cdir = File.expand_path('.')
     success = apply_measures(measures_dir, measures, runner, model)
+    Dir.chdir(cdir) # we need this because of Dir.chdir in HPXMLtoOS
```

Comment:
@shorowit Note these changes that need to be made in some test files, related to `Dir.chdir`.